src/main/java/org/spongepowered/api/world/explosion/Explosion.java

```diff
@@ -132,6 +130,15 @@ default double getKnockback() {
         return 1;
     }
 
+    /**
+     * Gets the {@link ResistanceCalculator} of the Explosion.
+     *
+     * @return The ResistanceCalculator or null if not supported by the implementation
+     */
+    default Optional<ResistanceCalculator> getResistanceCalculator() {
```

Comment:
I wrote the common comments before I saw this and the javadoc comments.

> or null if not supported by the implementation

I'd... rather not put that. Simply, `Optional#empty()` will indicate default behaviour.

Reply:
Could you clarify what you mean? I don't quite understand.

Reply:
Clarify the javadocs, don't refer to implementations.